ztp/policygenerator-kustomize-plugin/testPolicyGenTemplate/common-ranGen.yaml

```diff
@@ -22,15 +22,10 @@ spec:
       policyName: "ptp-sub-policy"
     - fileName: PaoSubscription.yaml
       policyName: "pao-sub-policy"
-      spec:
```

Comment:
Personally I think this section is better to be commented out instead of remove. As this provides an example on how we can override the default operator version.

Reply:
This file is just for unit tests - the ztp/ran-crd/policy-gen-template-ex.yaml file still has the channel override as an example.